schema/artist/statuses.js

```diff
@@ -10,61 +10,72 @@ const ArtistStatusesType = new GraphQLObjectType({
   fields: {
     artworks: {
       type: GraphQLBoolean,
+      resolve: ({ published_artworks_count }) => !!published_artworks_count,
```

Comment:
same thing, can change it if you like it better

On Fri, Jun 10, 2016 at 6:07 PM, mzikherman <notifications@github.com>
wrote:

> In schema/artist/statuses.js
> <https://github.com/artsy/metaphysics/pull/311#discussion_r66639384>:
>
> > @@ -10,61 +10,72 @@ const ArtistStatusesType = new GraphQLObjectType({
> >    fields: {
> >      artworks: {
> >        type: GraphQLBoolean,
> > +      resolve: ({ published_artworks_count }) => !!published_artworks_count,
>
> what's it mean for !! on an integer? Should it just be published_artworks_count
> > 0 (since it can never be nil)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/artsy/metaphysics/pull/311/files/efa6da403c6869ecd3fcf79a6c796d2d181c6920#r66639384>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ADBlTjvZe77Yz5G5z5o5O_h65y1Yjmw0ks5qKYvEgaJpZM4IzFFy>
> .
>



-- 

[A.]


Laura Brown

Artsy <http://artsy.net/>

laura@artsy.net <caroline@artsy.net>